view/frontend/web/js/view/payment/list/certificates-link.js

```diff
@@ -11,10 +11,41 @@
  * @copyright  Copyright (c) 2018 Avalara, Inc.
  * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  */
-define(['Magento_Checkout/js/model/totals'], function (totals) {
+define([
+    'Magento_Checkout/js/model/totals',
+    'certificatesSdk',
+    'jquery',
+    'Magento_Checkout/js/model/quote',
+    'Magento_Checkout/js/action/get-totals',
+    'Magento_Checkout/js/model/full-screen-loader',
+    'mage/storage',
+    'ClassyLlama_AvaTax/js/action/set-shipping-address'
+], function (totals, sdk, jQuery, quote, getTotalsAction, fullScreenLoader, storage, setShippingAddress) {
```

Comment:
Magento's standard alias for jQuery is `$`, so it breaks that convention to use `jQuery` in this function. Not a big deal, but I'd prefer `$`.